static/css/core.scss

```diff
@@ -218,6 +240,7 @@ section {
             gap: 8px;
             margin: 0;
             padding: 0;
+            font-weight: 500;
```

Comment:
Can you add font-weight:500 to links in the mobile menu and to the links on the bottom of the /about page? (Data Sources
, Disclaimers, Frequently Asked Questions, Blog)